Classes/Controller/AbstractController.php

```diff
@@ -162,4 +165,35 @@ protected function resolveView()
 	{
 		return null;
 	}
+
+
+	/**
+	 * Injects the Configuration Manager and is initializing the framework settings
+	 *
+	 * @param \TYPO3\CMS\Extbase\Configuration\ConfigurationManagerInterface $configurationManager Instance of the Configuration Manager
+	 */
+	public function injectConfigurationManager( ConfigurationManagerInterface $configurationManager )
+	{
+		// set ConfigurationManager and get settings like the FrontendConfigurationManager
+		$this->configurationManager = $configurationManager;
+		$this->settingsOrg = $this->configurationManager->getConfiguration( ConfigurationManagerInterface::CONFIGURATION_TYPE_SETTINGS );
+
+		$this->settings = $this->settingsOrg;
+
+		// load settings from template setup
+		$setup = $this->configurationManager->getConfiguration( ConfigurationManagerInterface::CONFIGURATION_TYPE_FULL_TYPOSCRIPT );
+		$extensionName = 'aimeos';
```

Comment:
perhaps this could be changed to more dynamic in the future. But for the moment I see no reason